services/softlayer_virtual_guest_test.go

```diff
@@ -166,6 +166,18 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
 		})
 	})
 
+	Context("#AttachEphemeralDisk", func() {
+		BeforeEach(func() {
+			fakeClient.DoRawHttpRequestResponse, err = common.ReadJsonTestFixtures("services", "SoftLayer_Product_Order_placeOrder.json")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("can attach a local disk without error", func() {
+			err := virtualGuestService.AttachEphemeralDisk(123, 25)
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
+
```

Comment:
for unit test, I'm thinking that we are using some fake http request to satisfy our testing scenario, so we suggest not put this testing scenario into the unit test